### PR TITLE
add parser and dependencies to app

### PR DESCRIPTION
**Pull request by poramental**
initial

### CODE REVIEW

File: src/main/java/com/vsuscheduleweb/entity/Group.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import java.util.List;
+
+// обьект группы
+
+
+@Setter
+@Getter
+@NoArgsConstructor
+@Accessors(chain = true)
+@ToString
```
Comment (SenyaDvaretski) [inline src/main/java/com/vsuscheduleweb/entity/Group.java]: Why you don't use @ Data and @ Entity? (same question in further entities)

File: src/main/java/com/vsuscheduleweb/entity/Group.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+public class Group {
+    String id;
+
+    List<Lesson> lessons = new ArrayList<>(); // общие пары
```
Comment (SenyaDvaretski) [inline src/main/java/com/vsuscheduleweb/entity/Group.java]: инглиш мазафака)

File: src/main/java/com/vsuscheduleweb/entity/Lesson.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+public class Lesson {
+    String startTime;
+
+    String endTime;
+
+    String auditorium;
+
+    String date;
+
+    String dayName;
```
Comment (SenyaDvaretski) [inline src/main/java/com/vsuscheduleweb/entity/Lesson.java]: For what?

File: src/main/java/com/vsuscheduleweb/entity/Teacher.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+// карточка преподователя
+@Setter
+@Getter
+@NoArgsConstructor
+@Accessors(chain = true)
+@ToString
+public class Teacher {
+    String firstName;
+    String lastName;
+    String Surname;
```
Comment (SenyaDvaretski) [inline src/main/java/com/vsuscheduleweb/entity/Teacher.java]: bad naming

**Review by SenyaDvaretski — COMMENTED** [review]
Where security is?)

**Comment by poramental** [comment]
security later

File: src/main/java/com/vsuscheduleweb/entity/Teacher.java (hunk 4 above)
Comment (poramental) [reply src/main/java/com/vsuscheduleweb/entity/Teacher.java]: +

Comment (SenyaDvaretski) [reply src/main/java/com/vsuscheduleweb/entity/Teacher.java]: hmmmmm

File: src/main/java/com/vsuscheduleweb/repositories/AppUserRepository.java [hunk 5]
```diff
@@ -0,0 +1,13 @@
+package com.vsuscheduleweb.repositories;
+
+
+import com.vsuscheduleweb.entity.AppUser;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Optional;
+import java.util.UUID;
+
+public interface AppUserRepository extends JpaRepository<AppUser,UUID> {
```
Comment (SenyaDvaretski) [inline src/main/java/com/vsuscheduleweb/repositories/AppUserRepository.java:12]: Why repository it is not a bean?)

File: src/main/java/com/vsuscheduleweb/security/roles/Permission.java [hunk 6]
```diff
@@ -0,0 +1,19 @@
+package com.vsuscheduleweb.security.roles;
+
+import lombok.Getter;
+
+@Getter
+public enum Permission {
+
+    HOTELS_READ("hotels:read"),
```
Comment (SenyaDvaretski) [inline src/main/java/com/vsuscheduleweb/security/roles/Permission.java]: may be rewrite everything with word "hotel"?)

**Pull request merged**